west.yml

```diff
@@ -93,8 +93,9 @@ manifest:
       revision: aef137b1af8aa7a0f43345c82459254b8832262e
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: e64c5f00d67cf858d1fdead7135f5d61de2a4c68
+      revision: pull/847/head
```

Comment:
PR for mcuboot synch https://github.com/zephyrproject-rtos/mcuboot/pull/38

Reply:
Noticed that, thanks. Just rebased the PR and the manifest update commit went away :+1: